ProcessMaker/processmaker
# Resolve Issue Preventing Project Permission Users from Managing Assets

This PR addresses the problem where users with only Project permissions were unable to create or manage assets within their projects.

## Solution

The solution involves the creation of a new PermissionMiddleware that enables a `can_any` permission check to verify if the user has any permissions, specifically the `create-projects` permission. This change allows Project permission users the ability to create and manage assets exclusively within their Projects.


## Changes
Implemented a new PermissionMiddleware to enable the `can_any` permission check for the `create-projects` permission, granting Project permission users the ability to create and manage assets within their Projects.

## How to Test
1. Create a new user.
2. Ensure that all project permissions are enabled by default for this user.
3. Log in as the newly created user.
4. Create a new Project.
5. From within the created project, add a new asset using the '+ asset' option.
6. Confirm that the user is able to successfully create and update new assets within their projects.

## Related Tickets & Packages
- [FOUR-11186](https://processmaker.atlassian.net/browse/FOUR-11186)
- [Projects PR](https://github.com/ProcessMaker/package-projects/pull/57)
- [Data Sources PR](https://github.com/ProcessMaker/package-data-sources/pull/335)
- Decision Engine PR

ci:next
ci:package-projects:task/FOUR-11186
ci:package-data-sources:task/FOUR-11186
ci:package-decision-engine:task/FOUR-11186

## Code Review Checklist


[FOUR-11186]: https://processmaker.atlassian.net/browse/FOUR-11186?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
